packages/light-apps/public/index.html

```diff
@@ -16,10 +16,10 @@
     form-action 'none';
     frame-src 'none';
     img-src 'self' data:;
-    manifest-src 'none';
+    manifest-src 'self';
     media-src 'none';
     object-src 'none';
-    script-src 'self' 'unsafe-inline';
+    script-src 'self' 'unsafe-inline' 'unsafe-eval';
```

Comment:
can't send transaction without wasm... I think we need to keep unsafe-eval unfortunately

Reply:
gave this a star in the hopes that some Googler somewhere sees it soon... https://bugs.chromium.org/p/chromium/issues/detail?id=948834&can=1&q=wasm-eval